testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -866,6 +878,181 @@ protected void checkInformationSchemaViewsForMaterializedView(String schemaName,
                 .containsAll("VALUES '" + viewName + "'");
     }
 
+    /**
+     * Test that reading table, column metadata, like {@code SHOW TABLES} or reading from {@code information_schema.views}
+     * does not fail when relations are concurrently created or dropped.
```

Comment:
Generally speaking do we want to "fix" this in all connectors? It's possible that the underlying system itself exhibits this behavior and we can do nothing except retries.

Reply:
Doing retries could be good way of "fixing".
If the underlying system is inherently problematic, disabling the test is OK.